src/js/controllers/homeController.js

```diff
@@ -112,5 +118,19 @@ export const homeScreen = async (state) => {
             }
         }
     });
+
+    document.querySelector(".home-top-icon").addEventListener('click',(e)=>{
```

Comment:
Can we name this better? Like edit-profile-icon or something more descriptive? Similarly, laputa is not descriptive enough, think about what would happen if we had multiple hidden dialog boxes

Reply:
This change involves changes in html and css. We will change it later.

Reply:
Created an issue

Next time, you should create an issue if you're not planning on resolving immediately and then resolve this conversation